site/content/docs/commands/task-run.md

```diff
@@ -39,6 +39,7 @@ Generally, the steps involved in task run are:
   --memory int                     Optional. The amount of memory to reserve in MiB for each task. (default 512)
   --resource-tags stringToString   Optional. Labels with a key and value separated with commas.
                                    Allows you to categorize resources. (default [])
+  --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated with commas. (default [])
```

Comment:
nit: same here, "with" -> "by"? And for `--resource-tags` and `--env-vars`, too?